### PR TITLE
UP-4743: Made cookie HTTP Only

### PR DESCRIPTION
**Pull request by bpowell**

### CODE REVIEW

File: uportal-war/src/main/java/org/apereo/portal/portlet/container/services/PortletCookieServiceImpl.java [hunk 1]
```diff
@@ -332,6 +332,7 @@ protected Cookie convertToCookie(IPortalCookie portalCookie, boolean secure) {
     	cookie.setComment(this.comment);
     	cookie.setMaxAge(this.maxAge);
     	cookie.setSecure(secure);
+        cookie.setHttpOnly(true);
```
Comment (ChristianMurphy) [inline uportal-war/src/main/java/org/apereo/portal/portlet/container/services/PortletCookieServiceImpl.java]: could this be adjusted to match the indentation of the surrounding code?

Comment (drewwills) [inline uportal-war/src/main/java/org/apereo/portal/portlet/container/services/PortletCookieServiceImpl.java]: It should match the indentation.  Our current standard (iirc) is 4 spaces, no tabs.

**Comment by drewwills** [comment]
Okay -- I took a closer look at this PR.  I have some findings.

1. This PR needs a JIRA ticket.  Please find or create one, edit the title, and provide a link in the description field
1. Please someone also fix the whitespace
1. Excellent find @ChristianMurphy;  I'm very happy you thought to look;  great work;  but there are a couple issues...
  * The Java change is a change to the PortletCookie, which is not the same kind of cookie that the JS is reading
  * The JS is trying to manipulate a kind of sidebar we no longer have;  nowhere does responder put out a `#portalSidebar` selector
1. We need to remove that JS file, and remove the references to that file (probably in the skin manifest)

**Comment by blamonet** [comment]
@drewwills issue created: https://issues.jasig.org/browse/UP-4743

**Comment by bpowell** [comment]
@drewwills I think I got everything you asked for. Let me know if I missed something.

**Pull request merged**